baybe/constraints/conditions.py

```diff
@@ -126,6 +159,13 @@ def _validate_tolerance(self, _: Any, value: float) -> None:  # noqa: DOC101, DO
                     f"or <= 0.0, but was {value}."
                 )
 
+    def generate_operator_function(self):
```

Comment:
So far, we've been using the `get_...` and `make_...` pattern, depending on whether objects already exist or need to be created. Also, shouldn't this be private? So I suggest renaming this into
```suggestion
    def _make_operator_function(self):
```

Reply:
changed